### DISCUSSION TITLE
Gutter icons option for codiumate like Jetbrains version

### DISCUSSION THREAD

**Discussion by nonoash** [Codium-ai/codiumai-vscode-release#172] [Ideas]
Hello,

i wanted to know if it was possible to have the option, just like the CodiumAI plugin in JB, for codiumate gutter icons next to functions?

I'm not a fan of extra added lines in the code as it is right now, because you lose a lot of vertical space but at the same time, the shortcut is useful and makes you remember its there. That's why i prefer the gutter icon option in JB, it solves this problem elegantly.

**Comment by almog-lv** [comment]
Hey @nonoash 
As far as I know, in VSCode it is not possible to add an icon on the side of the code, like in JB. In VSCode the extensions use code lens in most cases. I understand your pain about adding lines, that's why we gave the option to remove it and use the chat / right click on the code. I too hope it will be available also in VSCode.
